css/properties.json

```diff
@@ -4094,6 +4094,21 @@
     ],
     "status": "standard"
   },
+  "line-height-step": {
+    "syntax": " none | <length> ",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Fonts"
+    ],
+    "initial": "none",
+    "appliesto": "blockContainerElements",
+    "computed": "absoluteLength0ForNone",
+    "order": "perGrammar",
+    "status": "nonstandard"
```

Comment:
Sorry, but I would say definitely not "nonstandard". I guess "standard" is the default/standard setting I would set here. But possibly "experimental" (I'm not always clear on how we use that term, and I look at other usages of it to judge.)

Other properties that have "experimental": display-list, display-outside, font-variation-settings, initial-letter, max-block-size, offset, ... What do you think?